api/app/tests/apps/signals/test_tasks.py

```diff
@@ -304,3 +304,29 @@ def test_signal_receiver_child_status_afgehandeld(self):
         self.assertEqual(self.child_signal_1.status.state, AFGEHANDELD)
         self.assertEqual(self.child_signal_2.status.state, child_signal_2_state)
         self.assertNotEqual(self.child_signal_2.status.state, GEANNULEERD)
+
+    def test_task_parent_status_geannuleerd(self):
+        """
+        Parent Signal status to GEANNULEERD should trigger the children to status GEANNULEERD
+        """
+        parent_signal_state = self.parent_signal.status.state
+        child_signal_1_state = self.child_signal_1.status.state
+        child_signal_2_state = self.child_signal_2.status.state
+
+        status = Status.objects.create(state=GEANNULEERD, text='Test', _signal=self.parent_signal)
+        self.parent_signal.status = status
+        self.parent_signal.save()
+
+        with self.settings(FEATURE_FLAGS=self.test_feature_flags_enabled):
+            update_status_children_based_on_parent(signal_id=self.parent_signal.id)
```

Comment:
I would like to discuss the choice of not triggering based on an update to the parent signal. Not a blocker.